dss/storage/identifiers.py

```diff
@@ -57,6 +57,9 @@
 BUNDLES_PATTERN = f'/v1/bundles/{UUID_PATTERN}'
 BUNDLES_URI_REGEX = re.compile(BUNDLES_PATTERN)
 
+VERSIONED_TOMBSTONE_KEY_REGEX = re.compile(f"^(bundles)/({UUID_PATTERN}).({VERSION_PATTERN}).{TOMBSTONE_SUFFIX}$")
```

Comment:
is `bundles` supposed to have a value?

Reply:
also is this different from `DSS_BUNDLE_TOMBSTONE_REGEX ` ln 47?

Reply:
Not different. Thanks for catching that, will fix.